control/init.sh

```diff
@@ -17,8 +18,8 @@ function realpath() {
 
 
 if [ -z ${CI_TEST} ]; then
-    if hash python3.10 2>/dev/null && [[ $( source /etc/os-release; echo $VERSION_ID ) == 18.04 ]]; then
-        # if on 18.04 with 3.10 installed, use cchq-3.10
+    if ! $USE_SYSTEM_PYTHON && hash python3.10 2>/dev/null && [[ $( source /etc/os-release; echo $VERSION_ID ) == 18.04 ]]; then
```

Comment:
This will evaluate the contents of this variable as a command, right? I haven't seen that technique used before, and not sure if it's a security risk? In any case, seems like it would be more conventional to use string comparison
```suggestion
    if [[ $USE_SYSTEM_PYTHON == true ]] && hash python3.10 2>/dev/null && [[ $( source /etc/os-release; echo $VERSION_ID ) == 18.04 ]]; then
```

Reply:
That's a good point. Thanks for flagging.